playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -238,29 +238,22 @@
   any_errors_fatal: true
 
   pre_tasks:
+  - name: Load lib_openshift modules
```

Comment:
Right, we are not in a role so we don't have meta/main.yml.  Also, this has to be called in both pre_tasks and post_tasks to load the modules.